### DISCUSSION TITLE
How i get "Public Sponsor" in my github badge ?

### DISCUSSION THREAD

**Discussion by Kushal997-das** [Schweinepriester/github-profile-achievements#534] [Q&A] [answered]
How i get "Public Sponsor" in my github badge ? without Sponsoring someone ?

**Comment by yapryntsev** [comment]
It seems that for now it's not possible.

**Reply by Kushal997-das** [reply]
I see (:

**Comment by thehassantahir** [comment]
@Schweinepriester  well well, Nowadays people are here on Github to collect badges, lol
I can see a very low quality traffic for Github.

**Reply by Kushal997-das** [reply]
> @Schweinepriester well well, Nowadays people are here on Github to collect badges, lol I can see a very low quality traffic for Github.

I saw your profile most of the badges you earn by simple changing in `readme.md` file lol 😂

**Reply by thehassantahir** [reply]
Please attach screenshots 😉

**Comment by Kushal997-das** [comment, marked as answer]
Okey

**Comment by ArielMAJ** [comment]
@Kushal997-das 

I simply sponsored someone for 1 dollar. Feel free to sponsor me haha.

If you really don't wanna spend money, sponsor a friend and ask him to sponsor you. You could also create a secondary account and sponsor yourself (but idk if this goes against github's rules). For these options you'd need to set up your sponsor button.

If this answered your question, please mark this as answer.
